tests/test_utils.rs

```diff
@@ -37,3 +40,31 @@ pub fn continue_to_end(target: &LinuxTarget) {
         status => panic!("Status: {:?}", status),
     }
 }
+
+/// Turn the `pause` instruction inside the `breakpoint` function into a breakpoint.
+#[cfg(target_os = "linux")]
+#[allow(dead_code)]
+pub fn patch_breakpoint(target: &LinuxTarget, debuginfo: &RelocatedDwarf) {
+    // Test that `a_function` resolves to a function.
```

Comment:
This comment seems to be innacurate, probably should be something like
```suggestion
    // Get the address of the `breakpoint` function.
```